rewrite-gradle/src/main/java/org/openrewrite/gradle/plugins/AddPluginVisitor.java

```diff
@@ -91,7 +90,7 @@ private static Optional<String> findNewerVersion(String groupId, String artifact
     }
 
     private static MavenMetadata downloadMetadata(String groupId, String artifactId, List<MavenRepository> repositories, ExecutionContext ctx) throws MavenDownloadingException {
-        return new MavenPomDownloader(emptyMap(), ctx, null, null)
+        return new MavenPomDownloader(ctx)
```

Comment:
Like in other commits, let's not use the default repositories on the Gradle side as that's not accurate.